### PR TITLE
fix: mailto and tel links in g-link

### PR DESCRIPTION
**Pull request by sanscheese**
Have `mailto` and `tel` links also go to the attrs.href of the router-link.

### CODE REVIEW

File: gridsome/app/components/Link.js [hunk 1]
```diff
@@ -83,3 +83,7 @@ function isExternalLink (string) {
 
   return externalRE.test(string)
 }
+
+function isMailToLink (string) { return string.startsWith('mailto:') }
```
Comment (hjvedvik) [inline gridsome/app/components/Link.js]:
```suggestion
const isMailToLink = string => string.startsWith('mailto:')
```

File: gridsome/app/components/Link.js [hunk 2]
```diff
@@ -83,3 +83,7 @@ function isExternalLink (string) {
...
+
+function isTelLink (string) { return string.startsWith('tel:') }
```
Comment (hjvedvik) [inline gridsome/app/components/Link.js]:
```suggestion
const isTelLink = string => string.startsWith('tel:')
```

**Comment by sanscheese** [comment]
@hjvedvik came across `!!assets-loader` issue when testing with `mailto` links. So included a fix for this too (May cover - #740) 

Not sure if you have any shared utils across `app` and `lib` so i've kept them separate.

**Comment by hjvedvik** [comment]
Yes, `vue-remark` was failing because mailto links were transformed to `g-link`. I made `vue-remark` render them as normal links, but this fix would also fix it. Thanks :)

**Pull request merged**